.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -71,7 +75,7 @@ steps:
           mage build unitTest
         retry:
           automatic:
-           - limit: 3
+            - limit: 3
```

Comment:
All the unrelated formatting changes make this PR hard to review. Can we please only include changes related to RACE_DETECTOR / TEST_COVERAGE?

Reply:
> Should I raise another PR fixing the formatting?

Please do, thank you :bow: !